topics/yamlconfig.rst

```diff
@@ -261,35 +333,44 @@ automatically supplied default value that you get without the key word.
   VERBOSE           output logging level
-  rlimit            virtual memory limit per process
   runtime           maximum runtime in wall clock hours
   multistep         number of time step levels
-  shiftlevl         shift at most this number of step levels at one time
-  centerlevl        compute active centering below this multistep level
-  dynfracV          time step velocity coefficient (default: 0.01)
-  dynfracA          time step acceleration coefficient (default: 0.03)
-  dynfracS          time step crossing time for an externally set scale (default: 1.0)
-  dynfracS          time step potential/accerlation coefficient (default: 0.05)
-  use_cwd           the Node 0 home dir for the working dir on all nodes
-  posnsync          synchronizes multistep positions at each substep (default: true)
-  eqmotion          toggles phase space advance for use with externally supplied mapping.  On by default. 
-  global_cov        resets total center of velocity to zero if true 
   ldlibdir          directory containing loadable modules
   infile            input file for restart
   parmfile          parameter dump file
-  ratefile          initial processor rate file
   outdir            directory for output
   runtag            suffix or prefix used for labeling report files
   restart_cmd       command used to restart simulation on termination
   =============     ===========
 
+**Expert Parameters**
+
+These parameters should generally not need to be changed. We recommend that you
+ommit them from the YAML file, which will ensure that the default values will
+automatically be applied.
+  =============     ===========
+  Key Word          Explanation
+  =============     ===========
+  nbalance          number of steps between load balancing (use 0 for none)
+  rlimit            virtual memory limit per process
+  dbthresh          load balancing threshold (larger difference initiates balancing)
+  PFbufsz           particle ferry buffer size (the size of the internal MPI particle buffer)
+  ratefile          initial processor rate file
+  global_cov        resets total center of velocity to zero if true 
+  shiftlevl         shift at most this number of step levels at one time
+  centerlevl        compute active centering below this multistep level (set to 1/2 the multistep level, multistep is most conservative)
+  posnsync          synchronizes multistep positions at each substep (default: true)
+  use_cwd           the Node 0 home dir for the working dir on all nodes
+  dynfracV          time step velocity coefficient (default: 0.01)
+  dynfracA          time step acceleration coefficient (default: 0.03)
+  dynfracS          time step crossing time for an externally set scale (default: 1.0)
+  dynfracS          time step potential/accerlation coefficient (default: 0.05)
+  eqmotion          toggles phase space advance for use with externally supplied mapping.  On by default. 
+  =============     =========== 
```

Comment:
Good idea: *expert* parameters.

Reply:
The naming there is all Mike! He also helped a ton with figuring out what parameters should be considered 'expert'!

Let me know if there are any parameters that need to be moved to expert/not expert!

Reply:
The current partition looks right to me.  I go back and forth on the dynfrac parameters.  But think that they are more expert than not.  Some of the others are *super* expert!